TestResultSummaryService/parsers/TestBenchmarkParser.js

```diff
@@ -23,16 +21,10 @@ class TestBenchmarkParser extends Parser {
 
     // True iff all benchmarks and their corresponding iterations in the given run can be parsed
     static canParse( testName ) {
-        const {benchmarkName, benchmarkVariant} = this.getBenchmarkInfo(testName);
-        if (benchmarkName && benchmarkVariant && BenchmarkMetricRouter[benchmarkName]) {
-            if (!isNaN (parseInt(benchmarkVariant)) && BenchmarkMetricRouter[benchmarkName]["0"]) {// this works for TKG
-                return true;
-            } else if (BenchmarkMetricRouter[benchmarkName][benchmarkVariant]) {// this works for PerfNext
-                return true;
-            } else {
-                return false;
-            }
-        }     
+        const benchmarkParserkey = Utils.getBenchmarkParserKey(testName);
```

Comment:
Sorry, don't mean to be pedantic. If we are following camel case, then we should have upper case `K` in `benchmarkParserkey`. This applies to other usages as well.